spec/android_spec.rb

```diff
@@ -202,6 +202,16 @@
               to raise_error(StandardError, 'aapt not found - please create a symlink in $ANDROID_HOME/tools')
         end
 
+        it 'can return the Wifi mac address' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.